easybuild/easyconfigs/o/ORCA/ORCA-5.0.0-gompi-2021a-static.eb

```diff
@@ -0,0 +1,30 @@
+name = 'ORCA'
+version = '5.0.0'
+versionsuffix = '-static'
+
+homepage = 'https://orcaforum.kofo.mpg.de'
+description = """
+ORCA is a flexible, efficient and easy-to-use general purpose tool for quantum
+chemistry with specific emphasis on spectroscopic properties of open-shell
+molecules. It features a wide variety of standard quantum chemical methods
+ranging from semiempirical methods to DFT to single- and multireference
+correlated ab initio methods. It can also treat environmental and relativistic
+effects."""
+
+toolchain = {'name': 'gompi', 'version': '2021a'}
+
+# Static binaries of ORCA dynamically linking to OpenMPI
+sources = ['%%(namelower)s_%s_linux_x86-64_openmpi411_part%s.tar.xz' % (version.replace('.', '_'), p)
```

Comment:
I think we should start to use `%%(arch)s` in here since we are now providing better support for aarch64

Reply:
I made a custom template `%(orcaarch)s` because `%(arch)s` uses `x86_64` instead of `x86-64` :roll_eyes: